client/jetpack-cloud/sections/partner-portal/utils.ts

```diff
@@ -201,3 +201,18 @@ export function isJetpackBundle( product: APIProductFamilyProduct | string ) {
 	}
 	return product.family_slug === 'jetpack-packs';
 }
+
+/**
+ *	Checks if the license keys are compatible with multisite.
+ *
+ * @param licenseKeys
+ * @returns boolean indicating if the license keys are compatible with multisite
+ */
+export function isMultisiteCompatibleLicenseKeys( licenseKeys: string[] ): boolean {
```

Comment:
Ideally, we would get the license info and match it by product slugs, but this will cost us to delay the UI from rendering the site selections until we resolve the license info by fetching this data from the backend.

This function instead uses a license key prefix to determine what type of product we are trying to match.

Reply:
I think a better name for this function would be `areLicenseKeysMultisiteCompatible`.

Reply:
Yes that sounds better 👍